### PR TITLE
[BrokenLinksH2] Fixed broken links

### PR DESCRIPTION
**Pull request by kturetz**
Fixed broken links in iis-docs.

### CODE REVIEW

**Comment by kturetz** [comment]
**Global effort to fix broken links** 

@Rick-Anderson 
@ruslany 
@rmcmurray 
@wadepickett 

The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. I’d very much appreciate your timely review and feedback on the link changes I’ve suggested. Thanks!

**Review by wadepickett — COMMENTED** [review]
I understand given the situation with previous KB articles, why these are being moved to ` ` as the best option, of bad options to choose from.

For links to downloads that do not exist.  I don't see any benefit.  There is no way to find them in any roundabout way or 3rd party "wayback" site, etc using an old URI as a clue.  My recommendation is to find a working link, or eliminate the link, or eliminate the link but specify the name of the downloadable so at least it is understood historically what was used in the past.   Anyone have thoughts on this?

**Comment by kturetz** [comment]
> I understand given the situation with previous KB articles, why these are being moved to single quotes as the best option, of relatively bad options to choose from.
> 
> For links to downloads that do not exist. I don't see any benefit. There is no way to find them in any roundabout way or 3rd party "wayback" site, etc using an old URI as a clue. My recommendation is to find a working link, or eliminate the link, or eliminate the link but specify the name of the downloadable so at least it is understood historically what was used in the past. Anyone have thoughts on this?

That's been our approach, to comment out the link and leave it for historical reference.

**Comment by wadepickett** [comment]
Thanks for clarifying the pre-approval.  Sounds good.

File: iis/install/web-platform-installer/web-platform-installer-20-walkthrough.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ To get started, you need to download and install the Web Platform Installer. To
 
     By clicking **Install Now** and by using the Web PI, you agree to the [terms of use](https://go.microsoft.com/?linkid=9546162 "terms of use") for the Microsoft Web Platform Installer. The Web PI will contact Microsoft over the Internet to retrieve product information.
 
-    [Install Now](https://go.microsoft.com/?linkid=9588072 "Install Now")
+    [Install Now](https://www.microsoft.com/web/downloads/platform.aspx "Install Now")
```
Comment (wadepickett) [inline iis/install/web-platform-installer/web-platform-installer-20-walkthrough.md:27]:
```suggestion
    [Install Now](https://www.microsoft.com/web/downloads/platform.aspx)
```

**Review by wadepickett — APPROVED** [review]

**Comment by wadepickett** [comment]
No build errors.  There were lots of warnings. Looked through them, they appear to be the ones we expect that were already in the files, such as dupe H1, etc.
Merging.

**Pull request merged**